app/code/core/Mage/Core/Block/Abstract.php

```diff
@@ -582,8 +583,10 @@ public function getChildHtml($name = '', $useCache = true, $sorted = false)
             }
             return $out;
```

Comment:
Needs a `stop` before the return to keep the stack aligned.

Reply:
or removing the return $out in this place, as it is returned in the end